docs/settings.rst

```diff
@@ -75,9 +76,19 @@ URL endpoints where the middleware will be disabled. You can put your health che
 
 UUID_LENGTH
 -----------
-* **Default**: ``32``
+* **Default**: ``36``
```

Comment:
This seems like an unfortunate side-effect.  I guess it should be 36 for `str`, 32 for `hex`, and not used for `int`? Think changing the default like this would break my projects 🤔